cardinal.py

```diff
@@ -121,12 +123,34 @@
     # Get a logger!
     logger = logging.getLogger(__name__)
 
+    # Set the storage directory
+    storage_path = None
+    if config['storage'] is not None:
+        if config['storage'].startswith('/'):
+            storage_path = config['storage']
+        else:
+            storage_path = os.path.join(
+                os.path.dirname(os.path.realpath(sys.argv[0])),
+                config['storage']
+            )
+
+        logger.debug("Storage path set to %s" % storage_path)
+
+        if not os.path.exists(storage_path):
+            logger.info("Storage path does not exist, creating it..")
+            for folder in ['database', 'logs']:
```

Comment:
I actually considered this method too, since we might need to create more later. Awesome!

I'll get around to testing this out later today! Thanks again for the pull request! :+1: